macros/FirefoxOSDefaultAPISidebar.ejs

```diff
@@ -0,0 +1 @@
+REDIRECT <a class="redirect" href="/en-US/docs/Template:FirefoxOSAPISidebar">Template:FirefoxOSDefaultAPISidebar</a>
```

Comment:
This is the source for an MDN page that is a redirect to another file. If possible, we should not import these redirects.  One way to detect them is to look for content that starts with ``REDIRECT <a ``

I'm not sure how the live site currently handles these redirects. In other places, the pattern was not a redirect, but instead something like ``ifandmethod.ejs``:

```
<% /* Please use the up to date ifmethod template, this template currently redirects there. */ %>
<%- template('ifmethod', [$0, $1]) %>
```

or the more generic ``HTML.ejs``:

```
<%- template('Template:HTML:Element_Navigation', arguments) %>
```